_scripts/oauth_mysql.sh

```diff
@@ -5,5 +5,5 @@ check_mysql
 mysqlStarted=$?
 
 if [ "$mysqlStarted" ]; then
-  node ../fxa-oauth-server/bin/server.js 
+  node ../fxa-auth-server/fxa-oauth-server/bin/server.js 
```

Comment:
Due to updating `"cwd": "fxa-auth-server/fxa-oauth-server",` you are now running this within the `fxa-auth-server/fxa-oauth-server` directory, so I think you need to replace `../` with `../../` in two places in this file.

Reply:
(not going to attempt another push-my-own-fix on your PR though 😆)